dts/bindings/sensor/st,lps22hb-common.yaml

```diff
@@ -0,0 +1,61 @@
+# Copyright (c) 2021 STMicroelectronics
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  When setting the odr property in a .dts or .dtsi file you may include
+  lps22hb.h and use the macros defined there.
+
+  Example:
+  #include <zephyr/dt-bindings/sensor/lps22hb.h>
+
+  lps22hb: lps22hb@0 {
+    ...
+
+    odr = <LPS22HB_DT_ODR_200HZ>;
+  };
+
+include: sensor-device.yaml
+
+properties:
+  drdy-gpios:
+    type: phandle-array
+    description: |
+      DRDY pin
+
+      This pin defaults to active high when produced by the sensor.
+      The property value should ensure the flags properly describe
+      the signal that is presented to the driver.
+
+  odr:
+    type: int
+    default: 0
+    description: |
+      Specify the default output data rate expressed in samples per second (Hz).
+      The default is the power-on reset value.
+
+      - 0  # LPS22HB_DT_ODR_POWER_DOWN
```

Comment:
Use samples per second rather than the register value to match the Kconfig

Reply:
matched it